src/main/java/net/mcreator/ui/modgui/OverlayGUI.java

```diff
@@ -53,7 +54,7 @@ public OverlayGUI(MCreator mcreator, ModElement modElement, boolean editingMode)
 		JPanel pane5 = new JPanel(new BorderLayout(0, 0));
 
 		displayCondition = new ProcedureSelector(this.withEntry("overlay/display_condition"), mcreator,
-				"Display overlay in-game", ProcedureSelector.Side.CLIENT, true, VariableElementType.LOGIC,
+				L10N.t("elementgui.overlay.event_display_ingame="), ProcedureSelector.Side.CLIENT, true, VariableElementType.LOGIC,

```

Comment:
Remove the =

Reply:
Done! Sorry it was a typo but thx for noticing it!